### PR TITLE
Add rating controller AB#8889

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
# Fixes or Implements [AB#8889](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8889)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"? 

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

## Description

Adds the rating controller (missing service interface).

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### Steps to Reproduce:

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

File: Apps/WebClient/src/Server/Controllers/RatingController.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        /// <summary>
+        /// Saves the rating model.
+        /// </summary>
+        /// <param name="Rating">The rating to be saved.</param>
+        /// <returns>The saved rating wrapped in a request result.</returns>
+        /// <response code="200">Returns the saved rating json.</response>
+        /// <response code="401">the client must authenticate itself to get the requested response.</response>
+        /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
+        [HttpPost]
+        public IActionResult Save(object Rating) // TODO: update with Model.
```
Comment (FuriousLlama) [inline Apps/WebClient/src/Server/Controllers/RatingController.cs:62]: I've just added the model

File: Apps/WebClient/src/Server/Controllers/RatingController.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    /// <summary>
+    /// Web API to handle user ratings.
+    /// </summary>
+    [Authorize]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/api/[controller]")]
+    [ApiController]
+    public class RatingController
+    {
+        private readonly object ratingService; // TODO: Update with Interface.
```
Comment (leobcgov) [inline Apps/WebClient/src/Server/Controllers/RatingController.cs:41]: re. the development process, does this mean that we need to get another PR when the service is completed right?

Comment (tiagograf-AOT) [reply Apps/WebClient/src/Server/Controllers/RatingController.cs]: I've added the service interface!

File: Apps/WebClient/src/Server/Controllers/RatingController.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    /// <summary>
+    /// Web API to handle user ratings.
+    /// </summary>
+    [Authorize]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/api/[controller]")]
+    [ApiController]
+    public class RatingController
+    {
+        private IRatingService ratingService; // TODO: Update with Interface.
```
Comment (leobcgov) [inline Apps/WebClient/src/Server/Controllers/RatingController.cs]: minor: remove the // TODO:

File: Apps/WebClient/src/Server/Controllers/RatingController.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RatingController"/> class.
+        /// </summary>
+        /// <param name="ratingService">The injected rating service.</param>
+        public RatingController(
+            IRatingService ratingService) // TODO: Update with Interface.
```
Comment (leobcgov) [inline Apps/WebClient/src/Server/Controllers/RatingController.cs]: minor: remove the // TODO:

**Pull request merged**